### ISSUE TITLE
Creating second SwarmClient object with PSS endpoint produces ENOENT error

### ISSUE THREAD

**Issue by Chadsr** [MainframeHQ/erebos#74]
**System information**

OS & Version:Linux 4.19.4-1-MANJARO
Node Version: v11.1.0
Erebos Version: v0.5.1
Swarm Version: v0.3.6 & v0.3.7

**Expected behaviour**
Fire up two local Swarm nodes (on different ports) and successfully run the PSS example code found here: https://erebos.js.org/docs/examples#communication-between-two-nodes

**Actual behaviour**
Instantiating a second `SwarmClient` object with PSS endpoint fails with an ENOENT error:
```
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: connect ENOENT ws://localhost:8602
    at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1117:14)
Emitted 'error' event at:
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process.internalTickCallback (internal/process/next_tick.js:72:19)
```
It does not seem to be a problem with the nodes themselves. For instance running [nolash's pss-js example](https://github.com/nolash/pss-js) appears to function as expected.

**Steps to reproduce the behaviour**
- Spin up two Swarm nodes locally, both with websocket ports exposed*.
- Run the [PSS example code found in the Erebos documentation](https://erebos.js.org/docs/examples#communication-between-two-nodes) (with ports changed to match respective nodes).
- Observe `connect ENOENT` error when creating second `SwarmClient` object.

Switching which node is connected to first doesn't change this behaviour. Instantiating the first `SwarmClient` object is always successful and the second always fails.

*_I have tried spinning up nodes both with [swarm-test-cluster](https://github.com/janos/swarm-test-cluster) (Swarm v0.3.7) and docker image built from this repository (Swarm v0.3.6). The nodes themselves in both cases appear healthy and functioning as expected._

Edit:
A simple little example demonstrating that the error still occurs no matter which node is first:
```
import { SwarmClient } from '@erebos/swarm-node'

const shuffle = arr => arr.sort(() => Math.random() - 0.5)

const run = async () => {
  let wsPorts = [8601, 8602]
  shuffle(wsPorts)
  console.log(wsPorts)

  const alice = new SwarmClient({ pss: `ws://localhost:${wsPorts[0]}` })

  // Display Alice's public key
  await alice.pss.getPublicKey()
    .then(aliceKey => {
      console.log(`Alice: ${aliceKey}`)
    })

  // This will always error out (ENOENT), no matter which port
  const bob = new SwarmClient({ pss: `ws://localhost:${wsPorts[1]}` })

  // Display Bob's public key
  await bob.pss.getPublicKey()
    .then(bobKey => {
      console.log(`Bob: ${bobKey}`)
    })
}

run().catch(console.error)
```

**Comment by PaulLeCam** [comment]
Thanks for reporting this issue, it should be fixed now in v0.5.2, please try it out!

**Comment by Chadsr** [comment]
Yup, that did the trick.

Thanks for the speedy fix and release!
